h/accounts/views.py

```diff
@@ -574,6 +575,45 @@ def _update_subscription_data(request, subscription):
     request.session.flash(_('Changes saved!'), 'success')
 
 
+@view_config(attr='index', route_name='admin_users_index',
```

Comment:
Yeah I don't like the classes, but was just following the style of the rest of the views in this file. It does kind of make sense to group related views together in a file that has many